manager/assets/modext/widgets/resource/modx.panel.resource.js

```diff
@@ -315,6 +315,15 @@ Ext.extend(MODx.panel.Resource,MODx.FormPanel,{
             });
         }
     }
+    ,generateAliasRealTime: function(title) {
+        // check some system settings before doing real time alias transliteration
+        if (parseInt(MODx.config.friendly_alias_realtime, 10) && parseInt(MODx.config.automatic_alias, 10)) {
```

Comment:
That's an interesting approach to checking for a boolean value... I see it was that way before, but maybe worth improving this while you've made it into a nice function?

Reply:
All system settings has type `string` (field `value` of object [modSystemSetting](https://github.com/modxcms/revolution/blob/2.x/core/model/modx/mysql/modsystemsetting.map.inc.php#L38)). Everywhere boolean system settings checking as string: https://github.com/modxcms/revolution/blob/2.x/core/model/modx/modx.class.php#L2201

I think, we can't make this improvement within this PR.